R/pepfar-datapack.R

```diff
@@ -1,3 +1,5 @@
+PEPFAR_DATAPACK_PATH <- "pepfar_datapack_indicators_2024.csv"
```

Comment:
Just because I wanted to reference the same path from a test, so now if we change the path for any reason we'll only have to do that in one place. Instead of in the code and in the test.

Reply:
Thanks, sounds good. I added a commit a7a51252: 
* change `PEPFAR_DATAPACK_PATH` -> `PEPFAR_DATAPACK_FILENAME`
* add a comment in `outputs.R` to note it is defined in `pepfar-datapack.R`

Let me know if that all okay.